.github/workflows/release.yml

```diff
@@ -0,0 +1,39 @@
+name: Release
+
+on: 
+  push:
+    tags:
+      - '!player/'
+      - 'v*'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up node.js
+        uses: actions/setup-node@v3.3.0
+        with:
+          node-version-file: .nvmrc
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Test
+        run: npm test
+
+      - name: Build
+        run: gulp build-prod
+
+      - name: Publish
+        run: bash publish.sh
+        env:
+          NPM_DRY_RUN: false
```

Comment:
if this is hardcoded, why do we provide the default value?

Reply:
Because the `NPM_DRY_RUN` environment variable is used in the `publish.sh` script, and that env variable with the hardcoded value is also configured in TravisCI.

Reply:
How can you enable it?

Reply:
It's meant for local testing of the script, as far as I can tell. Again, I didn't write, add, or change that script, I just ported it over.